### ISSUE TITLE
Always Seen No!

### ISSUE THREAD

**Issue by ngr1998** [04x/TelegramView-Bypass#2]
even with good proxies
it always shows Seen NO!

**Comment by 04x** [comment]
Turn ON VPN And test again./thanks

**Comment by paul8909** [comment]
Im too always NO. Try use VPN, but not work(

**Comment by 04x** [comment]
let me check script. I will fix it if was a problem.

**Comment by paul8909** [comment]
ty, really looking forward to fixing

**Comment by paul8909** [comment]
Hey, i reinstall py2 any time and its work! But not all "seen ok" are counted( can make this script on py3? TY

**Comment by paul8909** [comment]
And need more userAgent for more profit, a think)

**Comment by 04x** [comment]
hmmm, New Bypass and new Update soon, Last And best version. I will release today or tomorrow!

**Comment by paul8909** [comment]
very much forward

**Comment by 04x** [comment]
Updated!, Done

**Comment by paul8909** [comment]
Thanx! I testing it, but not understand, what mean "cant get data view"? I use this proxy in browser and it loading page without problem.

**Comment by gdecker9317** [comment]
Hi, nice work! How to solve a problem when 12K proxy OK in Log, but the views are obtained 1K?
Thanks

**Comment by paul8909** [comment]
Not all proxy add views!

**Comment by gdecker9317** [comment]
No, i have in log more than 10K proxy - Seen OK, but counter track only +-1K
Mb i do some mistake? Pls can you help me?
I use sudo python GoTSeeN.py proxy.txt Post link

**Comment by gdecker9317** [comment]
i use calilinux

**Comment by 04x** [comment]
> 
> 
> Hi, nice work! How to solve a problem when 12K proxy OK in Log, but the views are obtained 1K?
> Thanks

Hi, Telegram will ban Proxies.

**Comment by udaykotian009** [comment]
Hi some one can help me in this ..am getting error..
thanks


Last login: Wed Mar  6 22:22:38 on console
Apples-MacBook-Pro:~ apple$ cd desktop
Apples-MacBook-Pro:desktop apple$ python GoTSeeN.py proxy.txt t.me/Goldsmarketresearch/105
  File "GoTSeeN.py", line 1
    {\rtf1\ansi\ansicpg1252\cocoartf1671
                                       ^
SyntaxError: unexpected character after line continuation character
Apples-MacBook-Pro:desktop apple$
